src/public/modules/forms/viewmodels/Form.class.js

```diff
@@ -1,3 +1,6 @@
+const _ = require('lodash')
+const formsg = require('@opengovsg/formsg-sdk')()
```

Comment:
we are initialising the SDK in production mode here, even if the code is running in dev/staging. this works fine for now because the SDK's `crypto.encrypt` function does not depend on FormSG's public key, but may lead to unexpected behaviour in future if this class has to call other SDK methods.